### PR TITLE
[DOC] Add Support page

### LINKED ISSUE #147: [DOC] Create a "get in touch" page in the documentation
We have [a section in the contributing guidelines](https://github.com/ME-ICA/tedana/blob/master/CONTRIBUTING.md#joining-the-conversation) on how to join the conversation, but we should also provide a page in [the ReadTheDocs site](https://tedana.readthedocs.io) for how to get in touch !

### PR DESCRIPTION
**Pull request by tsalo**
Closes #147. ~~We can wait to merge this until #142 is merged, but it's ready for review now.~~

Changes proposed in this pull request:
- Add "Support and Communication" page with links to Issues and NeuroStars.
- Move "Getting Involved" section from "Installation" page to "Contributing" page.
- Reorder pages on sidebar to better match fMRIPrep's example.

### CODE REVIEW

**Comment by emdupre** [comment]
Ok, I just merged #142 ! :rocket:

File: docs/support.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Support and Communication
+-------------------------
+All bugs, concerns and enhancement requests for this software can be submitted here: https://github.com/poldracklab/fmriprep/issues.
```
Comment (emdupre) [inline docs/support.rst]: This looks like an old link ! :laughing:

Comment (tsalo) [reply docs/support.rst]: Good catch! Should be fixed now.

File: docs/contributing.rst [hunk 2]
```diff
@@ -1,3 +1,19 @@
+Getting involved
```
Comment (emdupre) [inline docs/contributing.rst]: Can you pull this out ? It's a good reminder for me, though, to update `index.rst`...

Comment (emdupre) [reply docs/contributing.rst]: Discussed with @tsalo offline and removed this

**Pull request merged**